lesson07/assignment-alt/index.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <link rel="preconnect" href="https://fonts.gstatic.com">
+  <link href="https://fonts.googleapis.com/css2?family=Aleo:wght@300;400;700&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="css/style.css">
+  <script src="https://kit.fontawesome.com/2e8d1a037a.js" crossorigin="anonymous"></script>
+  <script src="https://unpkg.com/vue@next"></script>
+  <title>Document</title>
+</head>
+<body>
+  <main class="Main">
+      <nav class="Nav" id="hamburgerMenu">
+        <i class="Nav__menu fas fa-bars" v-on:click="toggleOn"></i>
+        <span class="Nav__title"><a class="Nav__title__a" href="index.html">Plant Project</a></span>
+        <i class="Nav__search-icon fas fa-search"></i>
+
+        <!-- Hamburger Menue-->
+        <div class="Nav__sliding-menu"  v-bind:style="toggle">
+          <ul class="Nav__sliding-menu__ul">
+            <i class="Nav__sliding-menu__ul__close-icon fas fa-times" id="nav_menu_close_icon" v-on:click="toggleOff"></i>
+            <li class="Nav__sliding-menu__ul__li"><a class="Nav__sliding-menu__ul__li__a" href="about.html">About Plant Project</a></li>
+            <li class="Nav__sliding-menu__ul__li"><a class="Nav__sliding-menu__ul__li__a" href="#">Sources Cited</a></li>
+            <li class="Nav__sliding-menu__ul__li"><a class="Nav__sliding-menu__ul__li__a" href="#">Contact Us</a></li>
+          </ul>
+        </div>
+
+        <!-- Tablet and Desktop Menu -->
+        <div class="Nav__desktop-menu">
+          <ul class="Nav__desktop-menu__ul">
+            <li class="Nav__desktop-menu__ul__li"><a class="Nav__desktop-menu__ul__li__a" href="about.html">About</a></li>
+            <li class="Nav__desktop-menu__ul__li"><a class="Nav__desktop-menu__ul__li__a" href="">Sources Cited</a></li>
+            <li class="Nav__desktop-menu__ul__li"><a class="Nav__desktop-menu__ul__li__a" href="">Contact Us</a></li>
+          </ul>
+        </div>
+      </nav>
+
+      <div class="Results">
+        <div class="Results__blurb">
+          <h3 class="Results__blurb__h3">Here's what we found near you:</h3>
+          <hr class="Results__blurb__hr">
+        </div>
+
+        <div class="Results__body" id="card-body">
+            <div v-bind:class="bodyClass" v-for="plant in plantBank">
```

Comment:
Perfect use case for a v-for loop. Might be cool if you can find a plant API that would allow you to load in the image and name data!